AcademyResidentInformationApi.Tests/V1/E2ETests/GetClaimantInformationById.cs

```diff
@@ -27,7 +27,7 @@ public async Task GetClaimantInformationByIdReturnsTheCorrectInformation()
             var academyId = $"{claimId}-{personRef}";
```

Comment:
```suggestion
```

Reply:
No, it returns academy Id. This would need another ticket I think.

Reply:
@LBHBomotoso and I have decided to have these changes that deal with the academy ID in a separate PR.